src/browser/modules/D3Visualization/components/DetailsPane.tsx

```diff
@@ -0,0 +1,283 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, { Component } from 'react'
+import { deepEquals, optionalToString } from 'services/utils'
+import {
+  StyledTokenRelationshipType,
+  StyledLabelToken,
+  StyledInspectorFooterRow,
+  StyledInspectorFooterRowListPair,
+  StyledInspectorFooterRowListKey,
+  StyledInspectorFooterRowListValue,
+  StyledInlineList,
+  StyledDetailsStatusBar,
+  StyledDetailsStatus,
+  StyledDetailsStatusContents,
+  StyledInspectorFooterRowListPairAlternatingRows,
+  StyledInspectorClipboardCopyAll,
+  StyledInspectorFooterRowListKeyValuePair
+} from './styled'
+import ClickableUrls from '../../../components/ClickableUrls'
+import numberToUSLocale from 'shared/utils/number-to-US-locale'
+import { StyledTruncatedMessage } from 'browser/modules/Stream/styled'
+import { Icon, Popup } from 'semantic-ui-react'
+import ClipboardCopier from 'browser-components/ClipboardCopier'
+
+const mapItemProperties = (itemId: string, itemProperties: any) => {
+  if (!itemProperties.length) {
+    return null
+  }
+  const clipboardCopy = (textToCopy: string) => (
+    <div style={{ marginLeft: 'auto' }}>
+      <ClipboardCopier textToCopy={textToCopy} iconSize={10} />
+    </div>
+  )
+
+  const itemPropertiesIncludingId = [
+    { key: '<id>', value: `${itemId}` },
```

Comment:
I put this to be in line with Figma sketches, but im not sure if `<id>` is correct...